src/content/smart-bulletins/036.md

```diff
@@ -0,0 +1,51 @@
+
+## Action
+
+Each Federal agency may, for Government owned or managed EVs used in accordance with 31 U.S.C 1344, “Passenger Carrier Use,” (1) allow for the vehicles to be charged at a commercial EV charging station and (2) pay for that charging with a GSA SmartPay charge card. The type of GSA SmartPay charge card to be used (purchase, travel, integrated and/or fleet) to complete the charging transaction must comply with agency policy. GSA SmartPay charge cards may also be used to pay for commercial rental car EV changing when an employee is in official travel status. Additional operational considerations for use of charge cards pursuant to the CHARGE Act are provided below:
+
+### GSA Leased Vehicles
+-  To enable the WEX fleet card for use at ChargePoint stations, the following options are available:
+    -  For agencies with at least one owned ChargePoint station, WEX accounts can be linked to ChargePoint RFIDs in the data portal or through the ChargePoint website. This action must be taken by the agency Fleet Manager(s).
+    - For agencies/drivers that want to charge at public ChargePoint stations contact GSA Fleet’s Alternative Fuel Vehicle Team at: gsafleetafvteam@gsa.gov and provide EV tag number(s) and delivery location(s). GSA Fleet will provide the driver with ChargePoint Radio Frequency IDentification (RFID) card(s) for the vehicle(s) linked to the vehicle’s associated GSA SmartPay fleet charge card. GSA Fleet EV commercial charging information resources include: a charging stations purchasing options and charging procedures at
+[gsa.gov/evse](http://gsa.gov/evse).
+- If an agency desires to use additional commercial charging station vendors beyond ChargePoint for GSA Fleet leased EVs, it must first contact the GSA Fleet Alternative Fuel Vehicle Team at: gsafleetafvteam@gsa.gov.
+### Agency-Owned Assets
```

Comment:
Agency-Owned Assets
Agencies are not limited to using ChargePoint charging stations for EV commercial charging of agency-owned assets. However, as stated above, ChargePoint is one of the few commercial charging vendors accepting WEX and Voyager at the time of the publication of this bulletin. For agencies with at least one owned ChargePoint station, a Fleet account may be set up with ChargePoint through their app or website. Further information on account setup for Fleet Card Managers can be found [here](https://docs.google.com/document/d/1tm6H3aQe-JqC9qcgkbx-dxLJdkFpB2ecLsFVi0Qm6pE/edit).
Voyager specific account setup information can be found on the ChargePoint site under the [Voyager Fleet Card Integration FAQs](https://www.chargepoint.com/solutions/voyager-faq/).
Most commercial charging stations do not have credit card terminals to swipe or “dip” cards, but some EVgo charges offer charge card terminals/readers. An account must first be established with the commercial charging network, typically through an app or website. The GSA SmartPay card account number and related information would be needed to establish the account. Most commercial charging networks offer payments on an individual and subscription basis. An RFID token (“fob”) or card is often required for use at the charging station to enable the transaction.
RFID cards/keyfobs for use at commercial EV charging stations are available on GSA Advantage or the open market where not available through GSA.
Agencies/organizations should confirm RFID/key fob acceptance with the desired networks before purchasing.